data/plot.py

```diff
@@ -31,22 +31,22 @@ def center_x(price):
         middle_price = (max_price - min_price) / 2 + min_price
 
         price = "%d" % max_price
-        draw.text((center_x(price), position_first[1]), price, font=font)
+        draw.text((center_x(price), position_first[1]), price, font=font, fill=fill)
         price = "%d" % middle_price
-        draw.text((center_x(price), (position_last[1] - position_first[1]) / 2 + position_first[1]), price, font=font)
+        draw.text((center_x(price), (position_last[1] - position_first[1]) / 2 + position_first[1]), price, font=font, fill=fill)
         price = "%d" % min_price
-        draw.text((center_x(price), position_last[1]), price, font=font)
+        draw.text((center_x(price), position_last[1]), price, font=font, fill=fill)
 
     @staticmethod
-    def caption(price, y, screen_width, font, draw):
-        draw.text((-1, y), "BTC", font=font)
+    def caption(price, x, y, screen_width, font, draw, fill=None):
+        draw.text((x, y), "BTC", font=font, fill=fill)
         price_text = "%.2f" % price
         text_width, _ = draw.textsize(price_text, font)
-        price_position = (((screen_width - text_width - 60) / 2) + 60, y)
-        draw.text(price_position, price_text, font=font)
+        price_position = (((screen_width - text_width - x) / 2) + x, y)
+        draw.text(price_position, price_text, font=font, fill=fill)
 
     @staticmethod
-    def candle(data, size=(100, 100), position=(0, 0), draw=None):
+    def candle(data, size=(100, 100), position=(0, 0), draw=None, fill_neg=None, fill_pos=None):
```

Comment:
The fill_neg should probably be "#000000" or black.

Reply:
Typically in the financial world black is used for positive while red is used for negative. Also, I didn't want to make assumptions of color values. The inkyWhat seems to have some constants that it uses and I don't know if CSS color names or hex codes would translate across all screens.